CHANGELOG.md

```diff
@@ -171,7 +171,13 @@ Quota metrics with specs of the form `users/<user>/read` and
 `users/<user>/write` are no longer exported by the Trillian binaries (as they
 lead to excessive storage requirements for Trillian metrics).
 
-### Fix Operation Loop Hang
+### Resilience improvements in `log_signer`
+
+#### Add timeout to sequencing loop
+
+Added a timeout to the context in the sequencing loop.
```

Comment:
Might be helpful to mention it's a 60 second timeout.

Reply:
Done